src/ocamlorg_web/lib/handler.ml

```diff
@@ -305,7 +305,19 @@ type package_kind =
   | Package
   | Universe
 
-let packages _req = Dream.html (Ocamlorg_frontend.packages [])
+let packages state _req =
+  let stats =
+    match Ocamlorg_package.packages_stats state with
+    | Some
+        { Ocamlorg_package.Packages_stats.nb_packages
+        ; nb_update_week
+        ; nb_packages_month
+        } ->
+      Some { Ocamlorg_frontend.nb_packages; nb_update_week; nb_packages_month }
```

Comment:
We don't have to create a new structure here I think, we can just pass the arguments to the function.

The less OCaml code in the frontend, the better, and using multiple arguments instead of a new type definition and a record is a bit lighter.

Reply:
That's two different records with the same fields. This is needed because the `ocamlorg_frontend` and `ocamlorg_package` libraries don't communicate. The same thing happens for the `package` type, this is probably done on purpose.
Making the libraries depend on each other might be worse because we might loose the ability to build the values. To really avoid the conversion, the best solution is to move the types in a separate library.

The record is needed because it is wrapped in an `option`. I think it's also easier to read that way.